DESIGN_DOCUMENT_TEMPLATE.md

```diff
@@ -0,0 +1,139 @@
+- [Using this Document](#using-this-document)
+	- [Feature Description](#feature-description-)
+	- [Architecture Diagram](#architecture-diagram)
+		- [API Diagram](#api-diagram)
+	- [Documenting your Feature](#documenting-your-feature)
+	- [Security Reviews](#security-reviews)
+
+# Using this Document
```

Comment:
What do you think of the idea of instruction like this in a template with  `<!-- -->` comments inline so it is hidden after creation?

Reply:
Gotcha